CHANGELOG.md

```diff
@@ -29,6 +29,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## [0.19.0] - 2024-01-??
+
+### Added
+
+- add `entityType` field in all extracted and merged models
+
+
```

Comment:
```suggestion
```

i think this needs to go under [[Unreleased] -> Added](https://github.com/robert-koch-institut/mex-common/blob/main/CHANGELOG.md?plain=1#L17)

version 19 and the date are only added once we do the actual release

Reply:
I set the date to today cause hopefully I can make the release today 🙏